src/main/resources/application.yml

```diff
@@ -76,7 +76,7 @@ notify:
 
 interventions-ui:
   locations:
-    sent-referral: "/referrals/{id}/confirmation"
+    sent-referral: "/service-provider/referrals/{id}"
```

Comment:
Sorry, getting out of touch again. Is this page only for providers?

Reply:
I guess what I'm pondering: should we have `locations.probation.sent-referral`, `locations.providers.sent-referral`?

Reply:
(Another not-really-relevant-right-now thought: this sounds like shared mapping; we could maintain a mapping in Kubernetes/Redis, seeded by the UI app, what the service could query)

Reply:
thanks for comments. yeh right now the only 'referral detail' page is targeted for providers. as a PP if you head to this page you see the 'assign to case worker' form. but the existing confirmation page doesn't give you any information about the referral at all. this is something we should loop around on and see if we need to make the referral landing page for PPs at this point too. maybe this will fall out of the action plan approval/EOSR stuff next sprint.

Reply:
thanks for explaining!